src/objects/SkinnedMesh.d.ts

```diff
@@ -1,15 +1,19 @@
-import { Geometry } from './../core/Geometry';
 import { Material } from './../materials/Material';
+import { MeshBasicMaterial } from './../materials/MeshBasicMaterial';
+
 import { Matrix4 } from './../math/Matrix4';
 import { Skeleton } from './Skeleton';
 import { Mesh } from './Mesh';
 import { BufferGeometry } from '../core/BufferGeometry';
 
-export class SkinnedMesh extends Mesh {
+export class SkinnedMesh<
+	TGeometry extends BufferGeometry,
```

Comment:
I prefer having `= BufferGeometry` here, depends on how [the discussion](https://github.com/mrdoob/three.js/pull/18720/files#r406096587) goes